java/src/jmri/jmrit/logix/Engineer.java

```diff
@@ -112,9 +114,9 @@ public void run() {
                         break;
                     }
                 } catch (InterruptedException ie) {
-                    log.error("InterruptedException " + ie);
+                    log.error("Engineer Interrupted at time" + ie);
```

Comment:
In that case, this message would be "Engineer Interrupted at timejava.lang.InterruptedException". Is that what is intended?